### PR TITLE
[Android] Implement chart/graph dependencies and ViewModel for Daily Reports screen

### PR DESCRIPTION
**Pull request by vikvelma**
T**ask ID: #013**
- Added chart/graph dependencies to gradle.
- Implemented viewModel class for `Daily Reports` screen.

### CODE REVIEW

File: Android/app/src/main/java/com/imobile3/groovypayments/ui/chart/PieChartViewModel.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PieChartViewModel extends ViewModel {
+
+    private ProductRepository mProductRepository;
```
Comment (vikvelma) [inline Android/app/src/main/java/com/imobile3/groovypayments/ui/chart/PieChartViewModel.java]: Not sure which repository to be used!!

File: Android/build.gradle [hunk 2]
```diff
@@ -4,11 +4,11 @@ buildscript {
     repositories {
         google()
         jcenter()
-        
+
```
Comment (HelloImKevo) [inline Android/build.gradle]: Can remove this extra newline, and the one below on line `22`

File: Android/app/src/main/java/com/imobile3/groovypayments/ui/chart/PieChartViewModel.java (hunk 1 above)
Comment (HelloImKevo) [reply Android/app/src/main/java/com/imobile3/groovypayments/ui/chart/PieChartViewModel.java]: 🤔 I think you could use the `CartRepository`, which would return a list of all the `Cart` models from the DB. The `Carts` represent orders that have been fulfilled for customers (and if you wanted further filtering, like for a Daily Report, I think there is a `CartEntity.DateCreated` field)

**Review by HelloImKevo — CHANGES_REQUESTED** [review]

File: Android/app/src/main/java/com/imobile3/groovypayments/ui/chart/PieChartViewModel.java (hunk 1 above)
Comment (vikvelma) [reply Android/app/src/main/java/com/imobile3/groovypayments/ui/chart/PieChartViewModel.java]: Cool, I assumed `CartRepository` only has the current order details!!

**Review by HelloImKevo — APPROVED** [review]
Looks great! I was able to patch this using:
```
git fetch git@github.com:vikvelma/SkeletonPaymentApp-Senior.git refs/pull/1/head:pr-daily-charts
```

**Pull request merged**